progression/src/main/resources/progression-migrations/V2__Create_items.sql

```diff
@@ -21,4 +21,7 @@ INSERT IGNORE INTO items (Material, Namespace, Keyname, Name, ModelData, Glow, h
 INSERT IGNORE INTO items (Material, Namespace, Keyname, Name, ModelData, Glow, hasUUID) VALUES ('ORANGE_GLAZED_TERRACOTTA', 'progression', 'speedy_bait', '<light_purple>Speedy Bait', 1, 0, 0);
 
 -- Rod
-INSERT IGNORE INTO items (Material, Namespace, Keyname, Name, ModelData, Glow, hasUUID) VALUES ('FISHING_ROD', 'progression', 'fishing_rod', '<yellow>Fishing Rod', 0, 0, 0);
\ No newline at end of file
+INSERT IGNORE INTO items (Material, Namespace, Keyname, Name, ModelData, Glow, hasUUID) VALUES ('FISHING_ROD', 'progression', 'fishing_rod', '<yellow>Fishing Rod', 0, 0, 0);
+
+-- Woodcutting-related
+INSERT IGNORE INTO items (Material, Namespace, Keyname, Name, ModelData, Glow, hasUUID) VALUES ('GLISTERING_MELON_SLICE', 'progression', 'tree_bark', '<aqua>Tree Bark', 0, 0, 0);
```

Comment:
Recommend giving this a CustomModelData of 1

Can be done in your resource pack like so

```
{
    "parent": "item/generated",
    "textures": {
        "layer0": "item/paper"
    },
    "overrides": [
		{"predicate": {"custom_model_data":2}, "model": "minecraft:item/clansmenu/alliances"}
     ]
}
```

Just an example